src/recycle_damn_iot.py

```diff
@@ -75,6 +79,18 @@ def parse_result(result):
             percentage = str("{:.2f}".format(percentage))
     return percentage
 
+def send_result(label, percentage):
+    bq = BigQueryClient()
+    
+    pred = Prediction()
+    pred.set_id(123)
```

Comment:
Is this always going to be static?

Reply:
Change incoming to use a timestamp instead

Reply:
Done